### PR TITLE
home UI & i18n

### PR DESCRIPTION
**Pull request by Bill-Haku**

### CODE REVIEW

File: HSRPizzaHelper/View/SysViews/AboutView.swift [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+//
+//  Created by Bill Haku on 2023/5/5.
+//
+
+import SwiftUI
+
+struct AboutView: View {
+    let appVersion = Bundle.main
+        .infoDictionary!["CFBundleShortVersionString"] as! String
+    let buildVersion = Bundle.main.infoDictionary!["CFBundleVersion"] as! String
```
Comment (CanglongCl) [inline HSRPizzaHelper/View/SysViews/AboutView.swift:13]: avoid force unwrap

File: HSRPizzaHelper/View/SysViews/ThanksView.swift [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+struct ThanksView: View {
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text("sys.thank.headline")
+                .font(.footnote)
+            Divider()
+            Group {
+                Text(
+                    "SwiftPieChart - Nazar Ilamanov\nhttps://github.com/ilamanov/SwiftPieChart"
```
Comment (CanglongCl) [inline HSRPizzaHelper/View/SysViews/ThanksView.swift:18]: We did not use Swift Pie Chart in this app

File: HSRPizzaHelper/View/SysViews/AboutView.swift (hunk 1 above)
Comment (CanglongCl) [reply HSRPizzaHelper/View/SysViews/AboutView.swift:13]: or use // swiftlint:disable ... to avoid warning

Comment (Bill-Haku) [reply HSRPizzaHelper/View/SysViews/AboutView.swift:13]: how to deal with it? convert `Any?` to `String`

Comment (CanglongCl) [reply HSRPizzaHelper/View/SysViews/AboutView.swift:13]: Just `(xx as? String) ?? "")´

**Review by CanglongCl — APPROVED** [review]

**Review by CanglongCl — APPROVED** [review]

**Pull request merged**